docs/meshstack.metadata-tags.md

```diff
@@ -249,19 +126,18 @@ Some Landing Zone assets like [GCP Cloud Functions](meshstack.gcp.landing-zones.
 | -------------------------------- | :--------------------------------------------------------------------------------------------------------------------------------------------------------------- |
 | `x-mesh-customer-identifier`     | meshCustomer Identifier                                                                                                                                          |
 | `x-mesh-project-identifier`      | meshProject identifier                                                                                                                                           |
-| `x-mesh-costcenter` *deprecated* | If available, ID of the CostCenter selected for this meshProject. Please use `x-mesh-tag-cost-center` or another [Payment Method tag](#payment-methods) instead. |
-| `x-mesh-tag-${format(tagName)}`  | metadata tags as defined in tag schema                                                                                                                           |
+| `x-mesh-costcenter` *deprecated* | If available, ID of the CostCenter selected for this meshProject. Please use `x-mesh-tag-cost-center` or another tag instead.                                    |
+| `x-mesh-tag-${format(tagName)}`  | metadata tags as defined in the tags screen in the admin area                                                                                                    |
 
 Headers for *metadata tags* are formatted to an http-header name by converting `camelCase` tag names into a dashed string i.e. `camel-case` and prefixing them with `x-mesh-tag-`.
 As a full example, a tag named `myCustomerTag` would be provided as an HTTP header with name `x-mesh-tag-my-customer-tag`.
 
 ## Tags in Cloud Tenants
 
-Certain cloud platforms allow you to tag resources in order to help organize them. It is possible to use the metadata tags from meshStack to control how these resources are tagged during project replication. The whole lifecycle of this tags is managed. This means tags on the cloud platform are updated and removed depending on the underlying metadata of the replicated meshProject. In order to properly track meshStack managed tags, every tag is usually prefixt with a tag prefix (e.g. the `meshstack_` prefix in the tag label `meshstack_costcenter:12345`).
+Certain cloud platforms allow you to tag resources in order to help organize them. It is possible to use the metadata tags from meshStack to control how these resources are tagged during project replication. The whole lifecycle of these tags is managed. This means tags on the cloud platform are updated and removed depending on the underlying metadata of the replicated meshProject. In order to properly track meshStack managed tags, every tag is usually prefixed with a tag prefix (e.g. the `meshstack_` prefix in the tag label `meshstack_costcenter:12345`).
```

Comment:
we should mention that we dont support resource tags natively but support fully tags on tenants